components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3446,6 +3443,39 @@ subroutine evaporate_sublimate_precip(qc_incld,qr_incld,nr_incld,t,pres, &
       qrevp = qrevp*ratio
    endif
 
+   else  ! orig_version
+
+   ! Determine temporary cloud fraction, set to zero if cloud water + ice is
+   ! very small.  This will ensure that evap/subl of precip occurs over entire
+   ! grid cell, since min cloud fraction is specified otherwise.
+   if (qc_incld + qitot_incld < 1.e-6_rtype) then
+      cld = 0._rtype
+   else
+      cld = lcldm
+   end if
+  
+   ! Only calculate if there is some rain fraction > cloud fraction
+   qrevp = 0.0_rtype
+   if (rcldm > cld) then
+      ! calculate q for out-of-cloud region
+      qclr = (qv-cld*qvs)/(1._rtype-cld)
+
+      ! rain evaporation
+      if (qr_incld.ge.qsmall) then
+         qrevp = epsr * (qclr-qvs)/ab
+      end if
+ 
+      ! only evap in out-of-cloud region
+      qrevp = -min(qrevp*(rcldm-cld),0._rtype)
+      qrevp = qrevp/rcldm
+   end if ! rcld>cld
+   if (qr_incld(i,k).gt.qsmall)  nrevp = qrevp*(nr_incld(i,k)/qr_incld(i,k))
+   ! No condensation, or evaporation of liquid
```

Comment:
this comment might be better phrased as "We are assuming macrophysics handles condensation/evaporation of qc and that there is no condensation of rain. Thus setting below values to zero"

Reply:
I guess when we switch entirely should we simply remove qccon, qcevp and qrcon from P3-SCREAM completely?

Reply:
I hadn't thought of removing them entirely, but yes we might as well.

Reply:
I'm happy to delete them all since carrying around and applying a bunch of variables set to 0 is kind of sloppy.  Once we do this it may be difficult for a P3-WRF person looking at the code to understand what happened to the evaporation and condensation variables they would expect.  I propose removing them entirely but having a comment in the microphysics tendencies declarations (or in the evaporation_sublimation... subroutine) that states that these processes have been removed because (see your comment above).  Does that sounds reasonable?

Reply:
Yeah, I think deleting them and leaving a comment is the way to go. Thanks for doing that.